ambry-api/src/main/java/com/github/ambry/account/Container.java

```diff
@@ -476,6 +480,24 @@ public static void setCurrentJsonVersion(short currentJsonVersion) {
     Container.currentJsonVersion = currentJsonVersion;
   }
 
+  /**
+   * Check if two containers' editable fields are same.
+   * @param container1 the {@link Container} to compare.
+   * @param container2 the {@link Container} to compare.
+   * @return {@code true} if two containers are equivalent in terms of editable fields. {@code false} otherwise.
+   */
+  public static boolean containerEditableFieldsAreSame(Container container1, Container container2) {
```

Comment:
This is tricky since "editable" concept is not well defined in OS ambry.
Also, should make this instance method taking only one Container argument to compare.

Reply:
Yeah, I changed it to instance method and renamed it to "isSameContainer"